pipeline.yml

```diff
@@ -19,4 +19,11 @@ urlRules:
   - type: forward
     conditions:
       - "/api/observations/*"
-    priority: 309
\ No newline at end of file
+    priority: 309
+tags:
+# wma:applicationId, wma:environment, and Name come for free via the ecs deployment pipeline
+  - wma:costCenter: "TBD"
```

Comment:
Seeing this written down, I'm not sure this needs to be an array. It could be just the key value pairs. :thinking: